x-pack/test/functional/apps/visualize/reporting.ts

```diff
@@ -65,21 +83,69 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
         await PageObjects.reporting.openPdfReportingPanel();
         expect(await PageObjects.reporting.isGenerateReportButtonDisabled()).to.be(null);
       });
+    });
 
-      it('downloaded PDF has OK status', async function () {
-        // Generating and then comparing reports can take longer than the default 60s timeout
-        this.timeout(180000);
+    describe('PNG reports: sample data created in 7.6', () => {
+      const reportFileName = 'tsvb';
+      let sessionReportPath: string;
 
-        await PageObjects.common.navigateToApp('dashboard');
-        await PageObjects.dashboard.loadSavedDashboard('Ecom Dashboard');
-        await PageObjects.reporting.openPdfReportingPanel();
+      before(async () => {
+        await kibanaServer.uiSettings.replace({
+          defaultIndex: 'ff959d40-b880-11e8-a6d9-e546fe2bba5f',
+        });
+
+        await esArchiver.load('x-pack/test/functional/es_archives/reporting/ecommerce_76');
+        await kibanaServer.importExport.load(
+          'x-pack/test/functional/fixtures/kbn_archiver/reporting/ecommerce_76.json'
+        );
+
+        log.debug('navigate to visualize');
+        await PageObjects.common.navigateToApp('visualize');
+        log.debug('load saved visualization');
+        await PageObjects.visualize.loadSavedVisualization(
+          '[K7.6-eCommerce] Sold Products per Day',
+          { navigateToVisualize: false }
+        );
+        log.debug('set time range');
+        await PageObjects.timePicker.setAbsoluteRange(
+          'Apr 15, 2022 @ 00:00:00.000',
+          'May 22, 2022 @ 00:00:00.000'
+        );
+
+        log.debug('open png reporting panel');
+        await PageObjects.reporting.openPngReportingPanel();
+        log.debug('click generate report button');
         await PageObjects.reporting.clickGenerateReportButton();
 
+        log.debug('get the report download URL');
         const url = await PageObjects.reporting.getReportURL(60000);
-        const res = await PageObjects.reporting.getResponse(url);
+        log.debug('download the report');
+        const reportData = await PageObjects.reporting.getRawScreenshotReportData(url);
+        sessionReportPath = await PageObjects.reporting.writeSessionReport(
+          reportFileName,
+          'png',
+          reportData,
+          REPORTS_FOLDER
+        );
+      });
+
+      after(async () => {
+        await esArchiver.unload('x-pack/test/functional/es_archives/reporting/ecommerce_76');
+        await kibanaServer.importExport.unload(
+          'x-pack/test/functional/fixtures/kbn_archiver/reporting/ecommerce_76.json'
+        );
+      });
+
+      // report is cut off
+      it.skip('PNG file matches the baseline image', async function () {
```

Comment:
This skipped test is incorrectly passing. The report PNG of a TSVB from the visualize app is mostly cut off. I believe this is due to work that still needs to go in from https://github.com/elastic/kibana/pull/131605